src/MOI_wrapper.jl

```diff
@@ -1,29 +1,16 @@
-using MathOptInterface
-const MOI = MathOptInterface
-const MOIU = MOI.Utilities
```

Comment:
MOIU is standard no?

Reply:
I really don't like it. `MOI.Utilities` is much clearer, and you very rarely need to use `MOIU` anyway.